client/client.go

```diff
@@ -733,3 +733,13 @@ func (client *Client) DebugGet(aspect string, result interface{}, params map[str
 	_, err := client.doSync("GET", "/v2/debug", urlParams, nil, nil, &result)
 	return err
 }
+
+type SystemRecoveryKeysResponse struct {
+	RecoveryKey      string `json:"recovery-key"`
+	ReinstallSaveKey string `json:"reinstall-save-key"`
```

Comment:
need to think a bit how to call this field

Reply:
maybe just ReinstallKey?

Reply:
yes